sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TableCatalog.java

```diff
@@ -311,4 +311,49 @@ default boolean purgeTable(Identifier ident) throws UnsupportedOperationExceptio
    */
   void renameTable(Identifier oldIdent, Identifier newIdent)
       throws NoSuchTableException, TableAlreadyExistsException;
+
+  /**
+   * Instantiate a builder to create a table in the catalog.
+   *
+   * @param ident  a table identifier.
```

Comment:
Nit: extra space

Reply:
Note: I guess you were trying to align the params?  But I think the other method javadocs here dont align if they are on the same line.

Reply:
Done